packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -49,10 +53,14 @@ const Layout = () => {
 
   const navItems = [
     { label: t('home'), to: '/' },
-    ...menu.map(({ label, contentId, type }) => ({
-      label,
-      to: type === PLAYLIST_TYPE.content_list ? contentListURL(contentId) : playlistURL(contentId),
-    })),
+    ...menu.map(({ label, contentId, type, custom }) => {
+      const translatedKey = custom?.[`label-${language}`];
+      const translatedLabel = translatedKey ? translatedKey : label;
+      return {
+        label: translatedLabel as string,
+        to: type === PLAYLIST_TYPE.content_list ? contentListURL(contentId) : playlistURL(contentId),
+      };
+    }),
```

Comment:
This can be simplified:

```suggestion
    ...menu.map(({ label, contentId, type, custom }) => ({
      label: custom?.[`label-${language}`] || label,
      to: type === PLAYLIST_TYPE.content_list ? contentListURL(contentId) : playlistURL(contentId),
    });
```